pkg/webui/console/views/application-api-keys-list/index.js

```diff
@@ -48,7 +48,7 @@ const ApplicationApiKeysList = () => {
   )
 
   return (
-    <div className="container container--xxl p-0">
```

Comment:
These `p-0`'s are deliberate to make correct for the tighter padding in list views. This also aligns with the screendesigns. Any reason they were removed?

Reply:
I included those because of this: https://github.com/orgs/TheThingsIndustries/projects/50?pane=issue&itemId=76149154. But maybe we can just set the horizontal padding to 0 and leave the vertical ones? @kschiffer

Reply:
I see. Ok let's look at this after the merge. We'll likely need some responsive classes there.